From b055ef84f33b8d2d98f8b59cb402648043cc2471 Mon Sep 17 00:00:00 2001
From: Tobias Hieta <tobias@hieta.se>
Date: Tue, 28 Nov 2023 09:52:28 +0100
Subject: [PATCH 20/35] Revert "[runtimes] Add missing test dependencies to
 check-all (#72955)"

This reverts commit e957e6dcb29d94e4e1678da9829b77009be88926.

The commit was reverted on main because of issues. We will not carry
this in the release branch for 17.x
---
 llvm/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/llvm/CMakeLists.txt b/llvm/CMakeLists.txt
index be0fc6c2ad7..79de9eb2e3e 100644
--- a/llvm/CMakeLists.txt
+++ b/llvm/CMakeLists.txt
@@ -1215,7 +1215,6 @@ if( LLVM_INCLUDE_TESTS )
   add_custom_target(test-depends
       DEPENDS ${LLVM_ALL_LIT_DEPENDS} ${LLVM_ALL_ADDITIONAL_TEST_DEPENDS})
   set_target_properties(test-depends PROPERTIES FOLDER "Tests")
-  add_dependencies(check-all test-depends)
 endif()
 
 if (LLVM_INCLUDE_DOCS)
-- 
2.44.0

